docs/source/notebook.rst

```diff
@@ -95,20 +95,23 @@ and open a web browser to the URL of the web application (by default,
 ``http://127.0.0.1:8888``).
 
 The landing page of the Jupyter notebook web application, the **dashboard**,
-shows the notebooks currently available in the notebook directory (by default,
-the directory from which the notebook server was started).
+shows the files and folders (subdirectories) in the notebook directory,  which by default is
+the directory from which the notebook server was started.
 
-You can create new notebooks from the dashboard with the ``New Notebook``
-button, or open existing ones by clicking on their name.  You can also drag
+You can create new text files, folders, terminals and notebooks from the dashboard with
+the dropdown menu attached to the ``New`` button.  You can also drag
 and drop ``.ipynb`` notebooks and standard ``.py`` Python source code files
-into the notebook list area.
+into the notebook list area, creating new entries on the dashboard (and corresponding
+new files in the server's file store).
 
 When starting a notebook server from the command line, you can also open a
 particular notebook directly, bypassing the dashboard, with ``jupyter notebook
 my_notebook.ipynb``. The ``.ipynb`` extension is assumed if no extension is
 given.
 
-When you are inside an open notebook, the `File | Open...` menu option will
+From the dashboard you can open existing notebooks in a new browser window by clicking on their name.
```

Comment:
From the dashboard ~~you can~~ open existing notebooks ~~in a new browser window~~ by double clicking on ~~their~~ the notebook's name.